js/processor.js

```diff
@@ -0,0 +1,33 @@
+(function() {
```

Comment:
What is the use of this file? It still references numerical conversion

Reply:
Oooooook I found the usage of this file.
Its main purpose is to show the description of the processor in the shaker, as shown here
![Capture d’écran 2021-01-11 à 4 19 39 PM](https://user-images.githubusercontent.com/54799490/104200374-df7f0280-5428-11eb-8b9b-85ba50834724.png)